tools-iuc/test-data.yaml

```diff
@@ -0,0 +1,3058 @@
+      sha: 54a3a43c801ff66f88f939edb984cf0fb5720255
+      unpack: false
+  doc: Test dataset operation_type|input_meryldb_02 for meryl.
+  license: MIT
+  size: 23375
+  used-by: meryl
+- data:
+    operation_type|input_meryldb_02:
+      filetype: meryldb
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/read-db.meryldb
+      sha: 54a3a43c801ff66f88f939edb984cf0fb5720255
+      unpack: false
+  doc: Test dataset operation_type|input_meryldb_02 for meryl.
+  license: MIT
+  size: 23375
+  used-by: meryl
+- data:
+    operation_type|input_meryldb_02:
+      filetype: meryldb
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/read-db.meryldb
+      sha: 54a3a43c801ff66f88f939edb984cf0fb5720255
+      unpack: false
+  doc: Test dataset operation_type|input_meryldb_02 for meryl.
+  license: MIT
+  size: 23375
+  used-by: meryl
+- data:
+    operation_type|input_meryldb_02:
+      filetype: meryldb
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/read-db.meryldb
+      sha: 54a3a43c801ff66f88f939edb984cf0fb5720255
+      unpack: false
+  doc: Test dataset operation_type|input_meryldb_02 for meryl.
+  license: MIT
+  size: 23375
+  used-by: meryl
+- data:
+    operation_type|input_meryldb_02:
+      filetype: meryldb
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/read-db.meryldb
+      sha: 54a3a43c801ff66f88f939edb984cf0fb5720255
+      unpack: false
+  doc: Test dataset operation_type|input_meryldb_02 for meryl.
+  license: MIT
+  size: 23375
+  used-by: meryl
+- data:
+    operation_type|input_meryldb_02:
+      filetype: meryldb
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/read-db.meryldb
+      sha: 54a3a43c801ff66f88f939edb984cf0fb5720255
+      unpack: false
+  doc: Test dataset operation_type|input_meryldb_02 for meryl.
+  license: MIT
+  size: 23375
+  used-by: meryl
+- data:
+    operation_type|child_reads:
+      filetype: null
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/child.fasta
+      sha: 5223a9704fb4547ed6f1afd7f5da3e2c03531ab2
+      unpack: false
+  doc: Test dataset operation_type|child_reads for meryl.
+  license: MIT
+  size: 1048576
+  used-by: meryl
+- data:
+    operation_type|paternal_reads:
+      filetype: null
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/paternal.fasta
+      sha: b00090b7a825d4c8e6048985d8e0cd58bd9bf95e
+      unpack: false
+  doc: Test dataset operation_type|paternal_reads for meryl.
+  license: MIT
+  size: 1048576
+  used-by: meryl
+- data:
+    operation_type|maternal_reads:
+      filetype: null
```

Comment:
seems to be a fasta problem? or a problem with multiple datatypes? Ah you are extracting it from ftype=?

Reply:
yes, exactly